File: Backend/Models/Diagnosis/Assessment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Models.Diagnosis
{

    public class Assessment
    {
        [Key]
        public int AssessmentId { get; set; }

        public int PatientId { get; set; }

        [Required]
        public string ChiefComplaint { get; set; }

        [Required]
        public string HistoryOfPresentIllness { get; set; }

        // body clicker comma separated string
        public string PainAssessment { get; set; }

        public int PainScale { get; set; }
    }
}

File: Frontend/CloudCure-Angular/src/app/AngularModels/Assessment.ts
export interface Assessment {
  id?: number;
  patientId: number;
  diagnosisId: number;
  chiefComplaint?: string;
  historyOfPresentIllness?: string;
  painAssessment?: string;
  painScale?: number;
  encounterDate?: Date;
  /*
  AssessmentId?: number;
  PatientId: number;
  ChiefComplaint: string;
  HistoryOfPresentIllness: string;
  PainAssessment?: string;
  PainScale?: number;
  */
}
